### DISCUSSION TITLE
Include boavizta.org work to evaluate embodied emissions

### DISCUSSION THREAD

**Discussion by AirLoren** [Green-Software-Foundation/software_carbon_intensity_data#8] [Ideas]
Hi,
I'm the co founder of Boavizta.org (a french non profit working group dedicated to the environmental footprint evaluation of digital services). As French electricity mix has a very low carbon intensity, we make a specific focus on embodied emissions and I suggest to evaluate the opportunity to integrate our work into SCI.
We wrote this first article focussed on server embodied emissions : https://boavizta.org/en/blog/empreinte-de-la-fabrication-d-un-serveur
And we are currently working on :
- networking embodied emissions
- storage embodied emissions
- a formula to evaluate Cloud services embodied emissions from the energy demand

**Comment by atg-abhishek** [comment]
Thank you for sharing this @AirLoren - we will take a look at it during this week's WG call.

**Comment by GadhuNTTDATA** [comment]
@AirLoren - This is very relevant since it is very difficult currently to measure M (Embodied Emission) for cloud resources. And the problem is over a few layers of abstraction:

1) Getting accurate LCA of the underlying hardware since there is a huge range across compute, storage and network hardware
2) Getting a TR(Time Reserved) value for virtual resources such as VMs that we purchase

One category of hardware that was not in the article was SAN storage. It will be a good area to explore further

**Comment by atg-abhishek** [comment]
@seanmcilroy29 to move this over to the SCI Open Data / Innovation WG folks to incorporate into our work.
